fdbclient/CommitTransaction.h

```diff
@@ -208,7 +208,7 @@ struct CommitTransactionRef {
 			if (ar.protocolVersion().hasReportConflictingKeys()) {
 				serializer(ar, report_conflicting_keys);
 			}
-			if (ar.protocolVersion().hasSpanContext()) {
+			if (ar.protocolVersion().version() >= (uint64_t)0x0FDB00B071000000LL) {
```

Comment:
Can we use something similar to `hasSpanContext()`? I introduced these two fields for the feature that moves private mutations to resolvers. Maybe add a function like `hasVersionVector()` or `hasResolverPrivateMutations()`, instead of hard coded constant `0x0FDB00B071000000LL`.

Reply:
Yes this is possible. I didn't want to use another 7.1 protocol feature that was unrelated, but was unsure whether it was worth it to add a new protocol feature. I'll make this update.